src/AltinnCore/Common/Models/RepositoryReleaseModel.cs

```diff
@@ -0,0 +1,9 @@
+namespace AltinnCore.Common.Models
+{
+    /// <summary>
+    /// RepositoryReleaseModel that represents a release in a Gitea repository
+    /// </summary>
+    public class RepositoryReleaseModel
```

Comment:
Is this empty class used for anything?

Reply:
Good catch, checking

Reply:
Done, deleted old code that was made obsolete after changes to the specification